test/hotspot/jtreg/containers/docker/TestMemoryAwareness.java

```diff
@@ -35,23 +35,26 @@
  *          jdk.jartool/sun.tools.jar
  * @build AttemptOOM sun.hotspot.WhiteBox PrintContainerInfo CheckOperatingSystemMXBean
  * @run driver ClassFileInstaller -jar whitebox.jar sun.hotspot.WhiteBox sun.hotspot.WhiteBox$WhiteBoxPermission
- * @run driver TestMemoryAwareness
+ * @run main/othervm -Xbootclasspath/a:whitebox.jar -XX:+UnlockDiagnosticVMOptions -XX:+WhiteBoxAPI TestMemoryAwareness
  */
 import jdk.test.lib.containers.docker.Common;
 import jdk.test.lib.containers.docker.DockerRunOptions;
 import jdk.test.lib.containers.docker.DockerTestUtils;
+import jdk.test.lib.sun.hotspot.WhiteBox;
```

Comment:
Should be:

```
import sun.hotspot.WhiteBox;
```

Reply:
Thank you for approving this PR, @jerboaa 
And thank you for pointing me to the error. I was in a hurry to make the SAP nightmare deadline. 
Learning? If you can't make it with diligence, don't do it. As always. 
import stmt is fixed.